download/_posts/2016-04-11-2.12.0-M4.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Scala 2.12.0-M4
+start: 11 April 2016
+layout: downloadpage
+release_version: 2.12.0-M4
+release_date: "April 11, 2016"
```

Comment:
I wasn't sure what to do here, since I didn't want to back-date this post, or forward-date the tag :-)